CHANGELOG.rst

```diff
@@ -6,6 +6,120 @@ Changelog
+    run separately, as it can take up to an hour to complete the entire operation for large
+    environments.
+
+    .. code-block:: bash
+
+        # in the container via ``docker exec`` or ``kubectl exec``:
+        python /app/bin/fix_submission_value_variable_missing_fields.py
+
+.. warning::
+
+    For the email and confirmation templates, and the registration backends, we changed
+    the way that the data is generated. In case of key conflicts between static, component,
+    and user-defined variables, the static variables will take precedence. Previously, the
+    component and user-defined variables would override the static variables. Our validation
+    guards against the use of keys that are already present in the static variables, but this
+    does not cover old forms and newly-added static variables.
+
+Detailed changes
+----------------
+
+**New features**
+
+* [:backend:`5268`] The registration plugins now support the ``children`` component type.
+* [:backend:`5095`] When authenticated via OpenID Connect (DigiD, eHerkenning, organization), upon
+  submission completion you are now logged out at the identity provider.
+* [:backend:`4951`] The map component in the summary PDF is now an image instead of the textual location
+  coordinates.
+* [:backend:`5269`] You can now use the ``children`` component data as source data for a
+  repeating group to provide additional information with the new "synchronize variables" logic
+  action type.
+* [:backend:`5575`] Added support for WMS tile layers in the map component, along
+  with import/export functionality.
+* [:backend:`5479`] The eIDAS (via OIDC) configuration is now simplified - you can specify
+  which claims can contain a BSN and/or Pseudo ID.
+* [:backend:`5060`] Redis Sentinel is now supported as high availability strategy for the background
+  jobs message broker.
+* [:backend:`5419`] Added prefill plugins for the new Yivi and eIDAS authentication backends.
+* [:backend:`5515`] Yivi Attribute groups now have a system-generated unique identifier.
+* [:backend:`5515`] You can now export and import Yivi attribute groups.
+* [:backend:`5253`] The BRT (grey, pastel, water) background tile layers are now available by
+  default in an Open Forms instance.
+* [:backend:`5251`] The identifier field from the map background tile layer is now automatically
+  populated from its label.
+
+* [:backend:`4879`] Finished the support for Worldline as payment provider:
+    - Support for Worldline's ``variant`` and ``descriptor`` fields.
+    - The merchant reference is generated by Open Forms, similar to the Ogone plugin.
+    - Ogone merchants are automatically migrated where possible.
+    - Webhook configuration (if configured in a older patch release) is automatically migrated.
+    - Moved the location in the admin of the feedback URL's used for the Worldline
+      payment webhooks to the webhook configuration page.
+    - Added a bulk action to migrate forms from Ogone to Wordline.
```

Comment:
```suggestion
* [:backend:`4879`] Finished the support for Worldline as payment provider:

    - Support for Worldline's ``variant`` and ``descriptor`` fields.
    - The merchant reference is generated by Open Forms, similar to the Ogone plugin.
    - Ogone merchants are automatically migrated where possible.
    - Webhook configuration (if configured in a older patch release) is automatically migrated.
    - Moved the location in the admin of the feedback URL's used for the Worldline
      payment webhooks to the webhook configuration page.
    - Added a bulk action to migrate forms from Ogone to Wordline.
```
the blank line is necessary for proper formatting, otherwise the outer level is formatted as bold text